org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/logger/AbstractLoggerSupport.java

```diff
@@ -40,6 +46,38 @@ public void setLogLevel(Logger logger, LogLevel level) {
     setLogLevel(assertNotNull(logger).getName(), level);
   }
 
+  protected void trackInitialState(String name) {
```

Comment:
add javadoc here for subclass-implementors? 
"Adds the current log-level of logger with given {@value name} to the initial state map"?

Reply:
Fixed in 6e0a26a3aa4980807138223d90d4956d76d2a13a